TAO/orbsvcs/orbsvcs/Naming/Storable_Naming_Context.cpp

```diff
@@ -1003,9 +1025,9 @@ TAO_Storable_Naming_Context::list (CORBA::ULong how_many,
       PortableServer::ObjectId_var id =
         PortableServer::string_to_ObjectId (poa_id);
 
-      this->poa_->activate_object_with_id (id.in (), svt.in());
-      CORBA::Object_var obj = this->poa_->id_to_reference (id.in ());
-      bi = CosNaming::BindingIterator::_narrow (obj.in());
+      this->poa_->activate_object_with_id (id.in (),
+                                           bind_iter);
+      bi = bind_iter->_this ();
```

Comment:
Isn't _this unavailable with CORBA/e? Vaguely remember having to update a lot of code in the past for that